src/NUnitFramework/framework/Constraints/Tolerance.cs

```diff
@@ -217,5 +217,16 @@ public bool IsUnsetOrDefault
         {
             get { return mode == ToleranceMode.Unset; }
         }
+
+        /// <summary>
+        /// Returns a string that represents the current object.
+        /// </summary>
+        /// <returns>
+        /// A string that represents the current object.
+        /// </returns>
+        public override string ToString()
+        {
+            return string.Format("{0} {1}", ((TimeSpan)amount).Milliseconds, "millisecond");
+        }
```

Comment:
Additionally, the code seems wrong - amount is an object with may or may not be a timespan. It might be a double, for example, used in comparing two doubles for equality. Anyway, please reset to HEAD.